pymongo/encryption_options.py

```diff
@@ -150,10 +152,33 @@ def __init__(
             outgoing commands. Set `bypass_query_analysis` to use explicit
             encryption on indexed fields without the MongoDB Enterprise Advanced
             licensed crypt_shared library.
+          - `encrypted_fields_map`: Map of collection namespace ("db.coll") to documents that
+            described the encrypted fields for Queryable Encryption. For example::
+
+                {
+                  "db.coll": {
```

Comment:
Update this to "db.encryptedCollection" to match the names you use in escCollection and friends below.